.github/workflows/lint.yml

```diff
@@ -4,7 +4,6 @@ on:
   pull_request:
     branches: [ master ]
   push:
-    branches: [ master ]
```

Comment:
I may miss something, but why should the CI be triggered for a push to a different branch than `master`?

Scenario in my head:
Create a branch to fix something. If during the development CI is needed, one could create a draft PR against the `master`, which triggers the CI. When the PR is merged, CI is triggered again, as it is a push on `master`.

Reply:
I see no problem to let lint run on all push. It will give me feedback if my feature branch is still compliant. And i can make style changes before i open a PR. Otherwise i have to open a PR, fix my style problems and update PR. It is cumbersome. Also the cost of running lint is like 20s totally.

Reply:
Okay, understood. Just as an FYI: you can run black on your code and it will fix the code automatically. You don't need to figure it out by yourself (flake8 mostly is happy after black).

In case of ARCHIE:
```
black *.py
```